index.js

```diff
@@ -2,12 +2,12 @@ const express = require("express");
 const mongoose = require("mongoose"); 
 const User = require("./models/user");
 const app = express(); 
-const userRouter = require("./routes");
+const artistRouter = require("./routes/artist");
+const mylistRouter = require("./routes/mylist");
```

Comment:
Might be better as `listRouter` instead of `myListRouter` but not critical to the task at hand

Reply:
Rectified